File: app/src/main/java/com/chad/baserecyclerviewadapterhelper/entity/Level1Item.java
package com.chad.baserecyclerviewadapterhelper.entity;

import com.chad.baserecyclerviewadapterhelper.adapter.ExpandableItemAdapter;
import com.chad.library.adapter.base.entity.AbstractExpandableItem;
import com.chad.library.adapter.base.entity.MultiItemEntity;

public class Level1Item extends AbstractExpandableItem<Person> implements MultiItemEntity {

  public String title;
  public String subTitle;

  public Level1Item(String title, String subTitle) {
    this.subTitle = subTitle;
    this.title = title;
  }

  @Override
  public int getItemType() {
    return ExpandableItemAdapter.TYPE_LEVEL_1;
  }

  @Override
  public int getLevel() {
    return 1;
  }
}

File: kotlinapp/src/main/java/com/allen/kotlinapp/entity/Level1Item.kt
package com.allen.kotlinapp.entity

import com.allen.kotlinapp.adapter.ExpandableItemAdapter
import com.chad.library.adapter.base.entity.AbstractExpandableItem
import com.chad.library.adapter.base.entity.MultiItemEntity

/**
 * 文 件 名: Level1Item
 * 创 建 人: Allen
 * 创建日期: 2017/6/13 15:10
 * 修改时间：
 * 修改备注：
 */

class Level1Item(var title: String, var subTitle: String) : AbstractExpandableItem<Person>(), MultiItemEntity {

    override fun getItemType(): Int {
        return ExpandableItemAdapter.TYPE_LEVEL_1
    }

    override fun getLevel(): Int {
        return 1
    }
}
